The tests fail on Darwin:

```
ciftilib> running tests
ciftilib> Test project /tmp/nix-build-ciftilib-1.6.0.drv-0/source/build
ciftilib>       Start  2: rewrite-little-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii
ciftilib>       Start  6: rewrite-little-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii
ciftilib>       Start 10: rewrite-little-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii
ciftilib>       Start 14: rewrite-little-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii
ciftilib>       Start 18: rewrite-little-ones.dscalar.nii
ciftilib>       Start  1: info-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii
ciftilib>       Start  4: datatype-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii
ciftilib>       Start  5: info-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii
ciftilib>  1/20 Test  #6: rewrite-little-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii .......***Failed    0.15 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib>       Start  7: rewrite-little-md5-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii
ciftilib>  2/20 Test  #2: rewrite-little-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii ........***Failed    0.16 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib>       Start  3: rewrite-little-md5-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii
ciftilib>  3/20 Test  #1: info-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii ..................***Failed    0.15 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib>       Start  8: datatype-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii
ciftilib>  4/20 Test  #4: datatype-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii ..............***Failed    0.15 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib>       Start  9: info-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii
ciftilib>  5/20 Test  #5: info-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii .................***Failed    0.15 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib>       Start 12: datatype-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii
ciftilib>  6/20 Test #18: rewrite-little-ones.dscalar.nii ............................................***Failed    0.16 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib>       Start 13: info-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii
ciftilib>  7/20 Test #10: rewrite-little-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii .......***Failed    0.17 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib>       Start 11: rewrite-little-md5-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii
ciftilib>  8/20 Test #14: rewrite-little-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii ...........***Failed    0.17 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib>       Start 15: rewrite-little-md5-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii
ciftilib>  9/20 Test  #7: rewrite-little-md5-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii ...***Failed    0.02 sec
ciftilib> CMake Error at /tmp/nix-build-ciftilib-1.6.0.drv-0/source/cmake/scripts/testmd5.cmake:1 (FILE):
ciftilib>   FILE MD5 failed to read file
ciftilib>   "little-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii": No such
ciftilib>   file or directory
ciftilib>       Start 16: datatype-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii
ciftilib> 10/20 Test  #3: rewrite-little-md5-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii ....***Failed    0.02 sec
ciftilib> CMake Error at /tmp/nix-build-ciftilib-1.6.0.drv-0/source/cmake/scripts/testmd5.cmake:1 (FILE):
ciftilib>   FILE MD5 failed to read file
ciftilib>   "little-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii": No such file
ciftilib>   or directory
ciftilib>       Start 17: info-ones.dscalar.nii
ciftilib> 11/20 Test #11: rewrite-little-md5-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii ...***Failed    0.01 sec
ciftilib> CMake Error at /tmp/nix-build-ciftilib-1.6.0.drv-0/source/cmake/scripts/testmd5.cmake:1 (FILE):
ciftilib>   FILE MD5 failed to read file
ciftilib>   "little-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii": No such
ciftilib>   file or directory
ciftilib>       Start 19: rewrite-little-md5-ones.dscalar.nii
ciftilib> 12/20 Test #15: rewrite-little-md5-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii .......***Failed    0.02 sec
ciftilib> CMake Error at /tmp/nix-build-ciftilib-1.6.0.drv-0/source/cmake/scripts/testmd5.cmake:1 (FILE):
ciftilib>   FILE MD5 failed to read file
ciftilib>   "little-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii": No such file or
ciftilib>   directory
ciftilib>       Start 20: datatype-ones.dscalar.nii
ciftilib> 13/20 Test #19: rewrite-little-md5-ones.dscalar.nii ........................................***Failed    0.02 sec
ciftilib> CMake Error at /tmp/nix-build-ciftilib-1.6.0.drv-0/source/cmake/scripts/testmd5.cmake:1 (FILE):
ciftilib>   FILE MD5 failed to read file "little-ones.dscalar.nii": No such file or
ciftilib>   directory
ciftilib> 14/20 Test  #8: datatype-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii .............***Failed    0.09 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib> 15/20 Test  #9: info-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii .................***Failed    0.09 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib> 16/20 Test #12: datatype-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii .............***Failed    0.09 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib> 17/20 Test #13: info-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii .....................***Failed    0.09 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib> 18/20 Test #16: datatype-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii .................***Failed    0.09 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib> 19/20 Test #17: info-ones.dscalar.nii ......................................................***Failed    0.09 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib> 20/20 Test #20: datatype-ones.dscalar.nii ..................................................***Failed    0.08 sec
ciftilib> Caught CiftiException: Cifti XML error: CIFTI element is missing the Version attribute
ciftilib> 0% tests passed, 20 tests failed out of 20
ciftilib> Total Test time (real) =   0.28 sec
ciftilib> The following tests FAILED:
ciftilib>         1 - info-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii (Failed)
ciftilib>         2 - rewrite-little-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii (Failed)
ciftilib>         3 - rewrite-little-md5-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii (Failed)
ciftilib>         4 - datatype-Conte69.MyelinAndCorrThickness.32k_fs_LR.dscalar.nii (Failed)
ciftilib>         5 - info-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii (Failed)
ciftilib>         6 - rewrite-little-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii (Failed)
ciftilib>         7 - rewrite-little-md5-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii (Failed)
ciftilib>         8 - datatype-Conte69.MyelinAndCorrThickness.32k_fs_LR.dtseries.nii (Failed)
ciftilib>         9 - info-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii (Failed)
ciftilib>        10 - rewrite-little-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii (Failed)
ciftilib>        11 - rewrite-little-md5-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii (Failed)
ciftilib>        12 - datatype-Conte69.MyelinAndCorrThickness.32k_fs_LR.ptseries.nii (Failed)
ciftilib>        13 - info-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii (Failed)
ciftilib>        14 - rewrite-little-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii (Failed)
ciftilib>        15 - rewrite-little-md5-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii (Failed)
ciftilib>        16 - datatype-Conte69.parcellations_VGD11b.32k_fs_LR.dlabel.nii (Failed)
ciftilib>        17 - info-ones.dscalar.nii (Failed)
ciftilib>        18 - rewrite-little-ones.dscalar.nii (Failed)
ciftilib>        19 - rewrite-little-md5-ones.dscalar.nii (Failed)
ciftilib>        20 - datatype-ones.dscalar.nii (Failed)
ciftilib> Errors while running CTest
error: builder for '/nix/store/zrkcfbz9496lwxk3myh32vyzrpypwxja-ciftilib-1.6.0.drv' failed with exit code 8
```

--- a/pkgs/development/libraries/science/biology/ciftilib/default.nix
+++ b/pkgs/development/libraries/science/biology/ciftilib/default.nix
@@ -1,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, boost
+, libxmlxx
+, pkg-config
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ciftilib";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Washington-University";
+    repo = "CiftiLib";
+    rev = "v${version}";
+    hash = "sha256-xc2dpMse4SozYEV/w3rXCrh1LKpTThq5nHB2y5uAD0A=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config ];
+  buildInputs = [ boost libxmlxx zlib ];
+
+  cmakeFlags = [ "-DCMAKE_CTEST_ARGUMENTS=--exclude-regex;'big|datatype-md5'" ];
+
+  doCheck = true;
+
+  meta = with lib; {
+    homepage = "https://github.com/Washington-University/CiftiLib";
+    description = "Library for reading and writing CIFTI files";
+    maintainers = with maintainers; [ bcdarwin ];
+    platforms = platforms.linux;
+    license = licenses.bsd2;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -33493,6 +33493,10 @@ with pkgs;
     inherit (llvmPackages) openmp;
   };
 
+  ciftilib = callPackage ../development/libraries/science/biology/ciftilib {
+    boost = boost16x;
+  };
+
   cmtk = callPackage ../applications/science/biology/cmtk { };
 
   clustal-omega = callPackage ../applications/science/biology/clustal-omega { };

